### PR TITLE
📝 Update README.md

### PR DESCRIPTION
**Pull request by mobily**
This PR adds small tweaks to the README.md file.

### CODE REVIEW

**Comment by mobily** [comment]
@googlebot I signed it!

File: README.md [hunk 1]
```diff
@@ -188,10 +188,8 @@ const Component = React.lazy(() => {
   switch (effectiveType) {
     case "3g":
       return import(/* webpackChunkName: "light" */ "./light.js");
-      break;
```
Comment (addyosmani) [inline README.md]: Changes above LGTM but I would like for us to maintain the break statements here. Could we bring these back?

Comment (mobily) [reply README.md]: @addyosmani thanks for the feedback! I've reverted the changes :)

**Comment by addyosmani** [comment]
Thanks!

**Comment by mobily** [comment]
@addyosmani thank you! 🎉

**Pull request merged**